File: java-spring/src/main/java/com/example/javaspring/util/GetColor.java
package com.example.javaspring.util;

import java.util.ArrayList;
import java.util.Arrays;

public class GetColor {

    private static final GetColor instance = new GetColor();

    public static String randomColor(ArrayList<String> list){
        if (list.isEmpty())
            list = new ArrayList<String>(Arrays.asList("Red", "Green", "Blue", "Black"));
        return list.get((int) Math.floor(Math.random() * list.size()));
    }
}

File: kotlin-spring/src/main/kotlin/com/example/kotlinspring/util/GetColor.kt
package com.example.kotlinspring.util

import kotlin.math.floor

object GetColor {
    fun randomColor(list: List<String>): String {
        if (list.isEmpty())
            return listOf("Red", "Green", "Blue", "Black")[floor(Math.random() * 4).toInt()]
        return list[floor(Math.random() * list.size).toInt()]
    }
}